Readme.md

```diff
@@ -126,7 +127,7 @@ dateFormat(now, "N");
 | `longTime`        | `h:MM:ss TT Z`                 | 5:46:21 PM EST           |
 | `isoDate`         | `yyyy-mm-dd`                   | 2007-06-09               |
 | `isoTime`         | `HH:MM:ss`                     | 17:46:21                 |
-| `isoDateTime`     | `yyyy-mm-dd'T'HH:MM:ss`        | 2007-06-09T17:46:21      |
+| `isoDateTime`     | `yyyy-mm-dd'T'HH:MM:sso`       | 2007-06-09T17:46:21+0700 |
```

Comment:
@fboes The isoDateTime mask is not changing as part of this, so the documentation for this should stay the same. Please revert this change :smile:

Reply:
Oh nevermind, the mask actually includes the `o` and you corrected an error in the documentation 😲  Nice job! :smile:

Reply:
@chase-manning You are right, my fix does not introduce this behaviour. Instead I believe that it is already there, but not documented,

Using `isoDateTime` I already get a result like `2020-12-05T10:13:29+0100`, and the code of the lib also states it to be `yyyy-mm-dd'T'HH:MM:sso` - so I thought to fix the documentation as well. ;)

```javascript
  dateFormat.masks = {
    default: "ddd mmm dd yyyy HH:MM:ss",
    shortDate: "m/d/yy",
    paddedShortDate: "mm/dd/yyyy",
    mediumDate: "mmm d, yyyy",
    longDate: "mmmm d, yyyy",
    fullDate: "dddd, mmmm d, yyyy",
    shortTime: "h:MM TT",
    mediumTime: "h:MM:ss TT",
    longTime: "h:MM:ss TT Z",
    isoDate: "yyyy-mm-dd",
    isoTime: "HH:MM:ss",
    isoDateTime: "yyyy-mm-dd'T'HH:MM:sso",
    isoUtcDateTime: "UTC:yyyy-mm-dd'T'HH:MM:ss'Z'",
    expiresHeaderFormat: "ddd, dd mmm yyyy HH:MM:ss Z",
  };
```

But if you want me to I remove that from the pull request, as it is not linked with the rest of the pull request.

Reply:
Yeah you were right! The docs were incorrect and you fixed them :smile: